openmls/src/messages/mod.rs

```diff
@@ -236,14 +236,16 @@ impl PathSecret {
         ciphersuite: Ciphersuite,
         public_key: &HpkePublicKey,
         group_context: &[u8],
-    ) -> HpkeCiphertext {
-        backend.crypto().hpke_seal(
-            ciphersuite.hpke_config(),
+    ) -> Result<HpkeCiphertext, LibraryError> {
+        hpke::encrypt_with_label(
```

Comment:
I think if you follow the callers of `encrypt` here up until `encrypt_path`, you can actually take an `&EncryptionKey` as input here, where you can just call `encrypt` and the implementation of `EncryptionKey` will add the right label for you. This would mirror what `decrypt` below is already doing.